### PR TITLE
chore(TDI-47181): change footer and header label

### PR DESCRIPTION
**Pull request by ozhelezniak-talend**
https://jira.talendforge.org/browse/TDI-47181

### CODE REVIEW

**Comment by ozhelezniak-talend** [comment]
Any suggestions on how we can change this behavior in a different way?
Without a change in the TCK-framework?

File: common-stream-io/common-io/src/main/java/org/talend/components/common/stream/format/OptionalLine.java [hunk 1]
```diff
@@ -23,23 +23,20 @@
...
     @Min(0)
     @ActiveIf(target = "active", value = "true")
     @Documentation("Number of lines.")
     private int size = 1;
 
+    public abstract boolean isActive();
+
     public int getSize() {
-        if (!this.active) {
+        if (!this.isActive()) {
```
Comment (undx) [inline common-stream-io/common-io/src/main/java/org/talend/components/common/stream/format/OptionalLine.java:39]: ```return this.isActive() ? this.size : 0;```

**Review by undx — APPROVED** [review]
LGTM

**Review by undx — APPROVED** [review]
LGTM

**Review by Tanya5091 — APPROVED** [review]
LGTM

**Pull request merged**